src/utils/ast/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface AragonContractFunction {
```

Comment:
I think is ok to have it here because this are specifics that the parser is doing for Aragon smart contract functions: https://github.com/aragon/buidler-aragon/blob/d80431bd367526767c30fe160ad0649047ea0620/src/utils/ast/parseContractFunctions.ts#L118